### PR TITLE
Module 1: Html and CSS Hello Microverse Project

### PR DESCRIPTION
**Pull request by alvinlouis29**
##In this project, I have built a simple webpage that displays the heading "Hello Microverse" in the color red.
##The main objective of this project is to show some of the best practices with Git and GitHub flow.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+# yarn v2
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.*
+
+# ignore the file named test.md
+.test.md
```
Comment (phelian23) [inline .gitignore]: - [x] I can see that you try to add the `test.md` file to the `.gitignore` file. But it should be `test.md` and not `.test.md`, and the node_modulus folder should be added as a whole. kindly fix that

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+ "This file should be ignored by git"
```
Comment (phelian23) [inline test.md:1]: - [x] Kindly ignore this file

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @alvinlouis29, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive README
- Clean code

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly set up the linters correctly following the guidelines listed [here](https://github.com/microverseinc/linters-config/tree/master/html-css).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (alvinlouis29) [reply .gitignore]: how can i add the node_modules as a whole

File: test.md (hunk 2 above)
Comment (alvinlouis29) [reply test.md:1]: the file mentioned above has been ignored

**Comment by alvinlouis29** [comment]
I also set up my linters well, that is why they are working well. i made sure they are setup well is that not the goal of this objective

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @alvinlouis29 , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — COMMENTED** [review]
Hello, @alvinlouis29 👋,

_This is an additional review as a result of the second opinion request._

Looks like the issue is resolved, and PR is already approved. 💯 

Keep rocking 🚀!

**Pull request merged**